suite-common/wallet-utils/src/contractsUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { parseAsset } from '@trezor/blockchain-link-utils/src/blockfrost';
+
```

Comment:
`tokenUtils` filename please

Reply:
It doesn't work anyway, now there's a circular dependency and type check is failing. I'm thinking what I should do, either move it somewhere else, or to get rid of some dependencies. Or to just introduce the hack with addresses

Reply:
Renamed